edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -0,0 +1,1349 @@
+2. **Adjust the number of scramble sets on the competition website**. This can be found at `https://www.worldcubeassociation.org/competitions/<competition-id>/events/edit`.
+   1. Typically, it is a good idea to create an extra scramble set for each round for each event in case it is needed on the day of the competition. It is a little more work to adjust it here and then remove them when preparing results, but it is easier to have extra scramble sets readily available in case a major incident occurs rather than having to generate new scramble sets on the fly.
+3. **Open TNoodle**, which launches a page in your web browser, and log in by authorizing it to use your WCA account and select the competition on the left.
+4. **Generate Scrambles**. Verify the number of scramble sets is correct and provide a password that will be used to protect the .zip file that is created. Click the "Generate Scrambles" button. The scramble generation process may take some time. When finished, the button will be named "Download Scrambles". Click it to download the .zip file to a location on your computer. **Do not lose this file** as you will need it when matching the scrambles to the results after the competition. 2. It is a smart idea to back this up to a secondary storage device or an online cloud service in case of hardware failure.
+5. **Get Files.** Using the password from the previous step, unarchive the .zip file. This file will contain files related to scramble generation and result submission:
+   1. `Interchange`, which is used when compiling results after the competition. This file contains the full scramble information for the competition in various formats.
+   2. `pdfs`, which contains the scrambles for the competition in a **password-protected** PDF file. These are the same files that are contained in the Computer Display .zip file (see below).
+   3. `Printing`, which contains the scrambles for the competition in **unprotected** PDF files and is designed for use when printing the scrambles onto paper.
+      1. As this folder contains all scrambles for the competition in an unprotected format, be very careful when handling this file. **This file must be inaccessible to scramblers using scramble display devices.**
+      2. Printing scrambles is highly discouraged under normal circumstances, due to the increased risk of a breach in scramble security. If you must print scrambles, make sure that they are always kept secure and only accessible to Delegates at all times.
+   4. `<competition-name> - Computer Display PDF Passcodes - SECRET`, which contains the passcodes to open the scramble set PDFs for each round. Only share this file with Delegates of the competition.
+      1. (optional) Input the contents of this into https://simonkellly.github.io/scramble-organizer/. You can then create a list in your note taking app of choice such as Google Keep and the passwords will be in order according to the competition schedule.
```

Comment:
I'm a bit concerned about including this link here, as this kind of tool is a potential security risk for the scramble passwords. It would be another thing if it were a desktop app that didn't require internet access, but it being a website that's not endorsed by the WRC/WST seems sketchy. @gregorbg do you have any thoughts about this one? If you think it's fine then we'll just leave it.

Reply:
I'd just get rid of 5.4.1 entirely

Reply:
For what it's worth, this tool is created by @simonkellly, another WST member. I understand your concerns with putting passwords into any random website, but this one in specific is from a trusted individual, and you can check the source code to see if it's doing anything sketchy with the passwords. This website in specific is not a risky vector.

Speaking as a Delegate, I really do not think I can understate the usefulness of this tool in a competition. It makes it substantially easier to manage and find passwords for scrambles. The whole point of this Handbook is to provide useful advice to Delegates, and the use of this tool is significantly useful. I strongly argue to keep.

Reply:
Oh, I totally missed that this was made by a WST member, haha. I'll resolve this then.